orbit/tools/build/build-windows.go

```diff
@@ -0,0 +1,265 @@
+package main
+
+// This tool builds Orbit binaries with versioninfo information.
+// https://learn.microsoft.com/en-us/windows/win32/menurc/versioninfo-resource
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"fmt"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/fleetdm/fleet/v4/orbit/pkg/constant"
+	"github.com/josephspurrier/goversioninfo"
+	zlog "github.com/rs/zerolog/log"
+)
+
+func main() {
+	// Input flags
+	flagVersion := flag.String("version", "0.0.1", "Version string")
+	flagCreateResource := flag.Bool("resource", false, "This is a bool flag to just create the resource.syso file and not build the binary")
+	flagIcon := flag.String("icon", "windows_app.ico", "Path to the icon file to embedded on the binary")
```

Comment:
Typo `Path to the icon file to embedded on the binary` -> `Path to the icon file to embed on the binary`

Reply:
fixed!